### PR TITLE
Update Readme: Integrity check failed for grommet

### PR DESCRIPTION
**Pull request by shaneelsharmaglaccount**
Added the resolution steps as a note in Read me --> Getting Stared section. This is based on the inputs provided by @halocline- Thanks ! 
I saw it on this link https://bytemeta.vip/repo/grommet/grommet-site/issues/206 , ..but couldn't find this added in readme

[Deploy Preview](https://deploy-preview-INSERT_PR_#_HERE--keen-mayer-a86c8b.netlify.app/)

#### What testing has been done on this PR?

In addition to the feature you are implementing, have you checked the following:

**General UX Checks**
- [ ] Small, medium, and large screen sizes
- [ ] Cross-browsers (FireFox, Chrome, and Safari)
- [ ] Light & dark modes
- [ ] All hyperlinks route properly

**Accessibility Checks**
- [ ] Keyboard interactions
- [ ] Screen reader experience
- [ ] Run WAVE accessibility plugin (Chrome)

**Code Quality Checks**
- [ ] Console is free of warnings and errors
- [ ] Passes E2E commit checks
- [ ] Visual snapshots are reasonable

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,6 +46,11 @@ yarn install
 
 Running 'yarn install' anywhere in the monorepo hierarchy will always install ALL the modules in the workspaces.
 
+Note: Sometimes you may get and error that says : "Integrity check failed for "grommet" (computed integrity doesn't match our records"...).
```
Comment (halocline) [inline README.md:49]:
```suggestion
Note: When installing, you may get and error saying "Integrity check failed for 'grommet' (computed integrity doesn't match our records...". 
```

File: README.md [hunk 2]
```diff
@@ -46,6 +46,11 @@ yarn install
 
 Running 'yarn install' anywhere in the monorepo hierarchy will always install ALL the modules in the workspaces.
 
+Note: Sometimes you may get and error that says : "Integrity check failed for "grommet" (computed integrity doesn't match our records"...). 
+Since grommet-site is referencing the latest stable branch of grommet. Any new commits added to the stable branch, the sha hash gets updated and becomes out of sync with the sha in the yarn.lock file.
```
Comment (halocline) [inline README.md:50]:
```suggestion
aries-site references the latest stable branch of grommet. Any new commits added to grommet's stable branch cause the sha hash to be updated and become out of sync with the sha in the yarn.lock file. 
```

File: README.md [hunk 3]
```diff
@@ -46,6 +46,11 @@ yarn install
 
 Running 'yarn install' anywhere in the monorepo hierarchy will always install ALL the modules in the workspaces.
 
+Note: Sometimes you may get and error that says : "Integrity check failed for "grommet" (computed integrity doesn't match our records"...). 
+Since grommet-site is referencing the latest stable branch of grommet. Any new commits added to the stable branch, the sha hash gets updated and becomes out of sync with the sha in the yarn.lock file. 
+
+To fetch the latest grommet stable, remove yarn.lock and clean yarn cache, then yarn install. For Example: rm yarn.lock && yarn cache clean && yarn install
```
Comment (halocline) [inline README.md:52]:
```suggestion
To fetch the latest grommet stable, remove yarn.lock and clean yarn cache, then yarn install. For example: `rm yarn.lock && yarn cache clean && yarn install`
```

**Review by halocline — COMMENTED** [review]
Thank you for your contribution!

The content looks great. I made a couple grammatical suggestions to help the content flow.

**Review by taysea — APPROVED** [review]
Thanks so much for the contribution @shaneelsharmaglaccount!

**Review by halocline — APPROVED** [review]

**Pull request merged**